__tests__/integration/api.test.js

```diff
@@ -192,6 +192,113 @@ describe("/api/articles", () => {
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 10 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual(testArticle);
+      });
+  });
+
+  test("PATCH: 200 increments an article's vote count by a negative amount and returns the article", () => {
+    const testArticle = { ...testData.articleData[0] };
+    testArticle.created_at = new Date(testArticle.created_at).toISOString();
+    testArticle.article_id = 1;
+    testArticle.votes -= 30;
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -30 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual(testArticle);
+      });
+  });
+
+  test("PATCH: 200 when inc_votes is 0, returns the article unchanged", () => {
+    const testArticle = { ...testData.articleData[0] };
+    testArticle.article_id = 1;
+    testArticle.created_at = new Date(testArticle.created_at).toISOString();
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 0 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual(testArticle);
+      });
+  });
+
+  test("PATCH: 200 ignores unnecessary fields in the request body", () => {
+    const testArticle = { ...testData.articleData[0] };
+    testArticle.created_at = new Date(testArticle.created_at).toISOString();
+    testArticle.article_id = 1;
+    testArticle.votes += 10;
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 10, article_id: 5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual(testArticle);
+      });
+  });
+
+  test("PATCH: 400 returns bad request when inc_votes is not a number", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "banana" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.status).toBe(400);
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+
+  test("PATCH: 400 returns bad request when inc_votes is missing from the request body", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.status).toBe(400);
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+
+  test("PATCH: 404 returns error when article_id doesn't exist", () => {
+    return request(app)
+      .patch(`/api/articles/${testData.articleData.length + 1}`)
+      .send({})
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.status).toBe(404);
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+
+  test("PATCH: 400 returns error when article_id is of invalid type", () => {
+    return request(app)
+      .patch("/api/articles/banana")
+      .send({})
```

Comment:
Same with this one, there's multiple things in the test that could be causing this error, this test should be sending a valid object